vaccine_feed_ingest/runners/ri/arcgis/normalize.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ak/arcgis/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["OBJECTID"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site = "arcgis"
+    runner = "ri"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "da57e8c8663048a2a9893c636fef63d0"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["USER_VaxType"].lower().split(",")
+
+    potentials = {
+        "pfizer-biontech covid-19 vaccine": schema.Vaccine(vaccine="pfizer"),
+        "moderna covid-19 vaccine": schema.Vaccine(vaccine="moderna"),
+        "janssen": schema.Vaccine(vaccine="janssen"),
+        "jjj": schema.Vaccine(vaccine="janssen"),
+        "varies": schema.Vaccine(vaccine="varies"),
```

Comment:
Please drop `varies` as a possible vaccine type - it's okay to leave this blank if the data isn't clear!

Additionally, from poking at the parsed data, it looks like there's some instances of `janssen (johnson & johnson) covid-19 vaccine`. Could you map that to `janssen` as well?

Reply:
Thanks. Done.